libshvchainpack/fuzz/fromChainpack.cpp

```diff
@@ -2,6 +2,9 @@
 
 extern "C" int LLVMFuzzerTestOneInput(const uint8_t* data, size_t size) {
     std::string err;
-    auto rpc_value = shv::chainpack::RpcValue::fromChainPack({reinterpret_cast<const char*>(data), size}, &err);
+    try {
+        auto rpc_value = shv::chainpack::RpcValue::fromChainPack({reinterpret_cast<const char*>(data), size}, &err);
```

Comment:
`shv::chainpack::RpcValue::fromChainPack()` should not throw if `err` is provided